app/app.go

```diff
@@ -707,7 +719,7 @@ func New(
 		capability.NewAppModule(appCodec, *app.CapabilityKeeper),
 		feegrantmodule.NewAppModule(appCodec, app.AccountKeeper, app.BankKeeper, app.FeeGrantKeeper, app.interfaceRegistry),
 		gov.NewAppModule(appCodec, app.GovKeeper, app.AccountKeeper, app.BankKeeper),
-		mint.NewAppModule(appCodec, app.MintKeeper, app.AccountKeeper),
+		mint.NewAppModule(appCodec, app.MintKeeper, app.AccountKeeper, nil),
```

Comment:
is there any specific reason why in `mint.NewAppModule` the BankKeeper is being passed in as `nil`, probably good to document it so that no one passes in a non nil BankKeeper later if there is a reason.

Reply:
That `nil` is actually an `InflationCalculationFn`:

`type InflationCalculationFn func(ctx Context, minter Minter, params Params, bondedRatio Dec) Dec`
> InflationCalculationFn defines the function required to calculate inflation rate during BeginBlock. It receives the minter and params stored in the keeper, along with the current bondedRatio and returns the newly calculated inflation rate. It can be used to specify a custom inflation calculation logic, instead of relying on the default logic provided by the sdk.